client/components/chat-button/index.tsx

```diff
@@ -6,21 +6,18 @@ import { useI18n } from '@wordpress/react-i18n';
 import classnames from 'classnames';
 import type { MessagingGroup } from '@automattic/help-center/src/hooks/use-messaging-availability';
 import type { FC } from 'react';
-import './style.scss';
```

Comment:
This is unrelated to the issue at hand, I'm just removing these styles as they are useless and not working anyway.